homeassistant/components/zyxelt50/config_flow.py

```diff
@@ -0,0 +1,58 @@
+"""Config flow for Zyxel T-50 integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({
+    vol.Required(CONF_HOST, default="192.168.1.1"): str,
+    vol.Required(CONF_USERNAME, default="admin"): str,
```

Comment:
You defined constants in `const.py`: DEFAULT_HOST, DEFAULT_USERNAME, what is the use for them?